src/Mutator/MutatorProfile.php

```diff
@@ -0,0 +1,207 @@
+        Mutator\Arithmetic\Decrement::class,
+        Mutator\Arithmetic\DivEqual::class,
+        Mutator\Arithmetic\Division::class,
+        Mutator\Arithmetic\Exponentiation::class,
+        Mutator\Arithmetic\Increment::class,
+        Mutator\Arithmetic\Minus::class,
+        Mutator\Arithmetic\MinusEqual::class,
+        Mutator\Arithmetic\ModEqual::class,
+        Mutator\Arithmetic\Modulus::class,
+        Mutator\Arithmetic\MulEqual::class,
+        Mutator\Arithmetic\Multiplication::class,
+        Mutator\Arithmetic\Plus::class,
+        Mutator\Arithmetic\PlusEqual::class,
+        Mutator\Arithmetic\PowEqual::class,
+        Mutator\Arithmetic\ShiftLeft::class,
+        Mutator\Arithmetic\ShiftRight::class,
+    ];
+
+    const BOOLEAN = [
+        Mutator\Boolean\FalseValue::class,
+        Mutator\Boolean\LogicalAnd::class,
+        Mutator\Boolean\LogicalLowerAnd::class,
+        Mutator\Boolean\LogicalLowerOr::class,
+        Mutator\Boolean\LogicalNot::class,
+        Mutator\Boolean\LogicalOr::class,
+        Mutator\Boolean\TrueValue::class,
+    ];
+
+    const CONDITIONAL_BOUNDARY = [
+        Mutator\ConditionalBoundary\GreaterThan::class,
+        Mutator\ConditionalBoundary\GreaterThanOrEqualTo::class,
+        Mutator\ConditionalBoundary\LessThan::class,
+        Mutator\ConditionalBoundary\LessThanOrEqualTo::class,
+    ];
+
+    const CONDITIONAL_NEGOTIATION = [
+        Mutator\ConditionalNegotiation\Equal::class,
+        Mutator\ConditionalNegotiation\GreaterThanNegotiation::class,
+        Mutator\ConditionalNegotiation\GreaterThanOrEqualToNegotiation::class,
+        Mutator\ConditionalNegotiation\Identical::class,
+        Mutator\ConditionalNegotiation\LessThanNegotiation::class,
+        Mutator\ConditionalNegotiation\LessThanOrEqualToNegotiation::class,
+        Mutator\ConditionalNegotiation\NotEqual::class,
+        Mutator\ConditionalNegotiation\NotIdentical::class,
+    ];
+
+    const FUNCTION_SIGNATURE = [
+        Mutator\FunctionSignature\PublicVisibility::class,
+        Mutator\FunctionSignature\ProtectedVisibility::class,
+    ];
+
+    const NUMBER = [
+        Mutator\Number\DecrementInteger::class,
+        Mutator\Number\IncrementInteger::class,
+        Mutator\Number\OneZeroInteger::class,
+        Mutator\Number\OneZeroFloat::class,
+    ];
+
+    const OPERATOR = [
+        Mutator\Operator\Break_::class,
+        Mutator\Operator\Continue_::class,
+        Mutator\Operator\Throw_::class,
+    ];
+
+    const RETURN_VALUE = [
+        Mutator\ReturnValue\FloatNegation::class,
+        Mutator\ReturnValue\FunctionCall::class,
+        Mutator\ReturnValue\IntegerNegation::class,
+        Mutator\ReturnValue\NewObject::class,
+        Mutator\ReturnValue\This::class,
+    ];
+
+    const SORT = [
+        Mutator\Sort\Spaceship::class,
+    ];
+
+    const ZERO_ITERATION = [
+        Mutator\ZeroIteration\Foreach_::class,
+    ];
+
+    const DEFAULT = [
+        '@arithmetic',
```

Comment:
can we move these values to the constants?

Reply:
I'm not sure what you mean.

Do you mean something like 
```php
const ARITHMETIC_PROFILE = '@arithmetic';
```